packages/devtools_app/lib/src/logging/logging_controller.dart

```diff
@@ -656,17 +658,28 @@ class LogData {
 
   final RemoteDiagnosticsNode node;
   String _details;
-  Future<String> detailsComputer;
+  Future<String> Function() detailsComputer;
+
+  static const JsonEncoder prettyPrinter = JsonEncoder.withIndent('    ');
```

Comment:
why indent 4? use indent 2 and a fixed sized font instead. There is some similar code in the json display in the timeline.

Reply:
We don't have any monospaced fonts bundled yet.  I've left a TODO.